back-end/Models/comments/comment.js

```diff
@@ -0,0 +1,51 @@
+const connection = require("../../db");
+
+//Add new comment
+const addComment = (req, res) => {
+  const comment = req.body;
+  comment.user_id = null;
+  comment.post_id = null;
+  const query = `INSERT INTO comment (comment, thumbs_up) VALUES (?, ?)`;
+  const data = [comment.comment, comment.thumbs_up];

```

Comment:
Would be better if you would use destructuring so you don't repeat adding `comment.keyName`
Why are you setting the `user_id` and `post_id` to null?